package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4715,6 +4915,7 @@ def _matches_unpickling(self, anchor_name, n_atoms, fname, trajname):
             return False
 
 
+@deprecate(message=_SIXTEEN_DEPRECATION)
```

Comment:
Yeah, it doesn't make sense at the moment in 363, so best to deprecate and then not remove it at all if it ends up being needed.